File: TPraticas/Aula11.a/codigofonte/LOverflow3.java


import java.util.Scanner;

public class LOverflow3 {
  public static void main(String[] args) {

    Scanner scan = new Scanner(System.in);
    int[] vals = new int[10];

    System.out.println("Quantos valores quer guardar no array? ");
    int count = scan.nextInt();

    for (int i = 0; i < count; i++ ) {
      vals[i] = count-i;
    }

    System.out.println("Que valor deseja recuperar? ");
    int which = scan.nextInt();

    System.out.println("O valor é " +vals[which]);
  }
}

File: TPraticas/Aula11.a/codigofonte/LOverflow3.cpp
/* 
 * File:   Overflow3.cpp
 * Author: jepm
 *
 * Created on November 13, 2014, 12:40 PM
 */

#include <iostream>
using namespace std;

int main(void)
{
  int vals[10];
  size_t count;
  size_t which;

  cout << "Quantos valores quer guardar no array? ";
  cin >> count;

  for (size_t i = 0; i < count; i++) {
    vals[i] = count-i;
  }

  cout << "Que valor deseja recuperar? ";
  cin >> which;

  cout << "O valor é " << vals[which] << endl;

  return 0;
}
